### PR TITLE
Fix GCP prerequisites

### PR DESCRIPTION
**Pull request by rcaballeromx**
Addressing feedback gotten post-merge.

### CODE REVIEW

File: compute/admin_guide/vulnerability_management/vm_image_scanning.adoc [hunk 1]
```diff
@@ -150,10 +150,9 @@ compute.subnetworks.use
 compute.subnetworks.useExternalIp
 ----
 
-* Verify that your project has a service account with the `Cloud KMS CryptoKey Decrypter` role or equivalent.
-
-* Verify that the **Compute Engine Service Agent** xref:https://cloud.google.com/iam/docs/service-agents[Google-managed service account] ends with `compute-system.iam.gserviceaccount.com`
-
+* Verify that the Compute Engine Service Agent service account in the target image project has the `Cloud KMS CryptoKey Decrypter` role or equivalent. 
++
+This built-in service account ends with `compute-system.iam.gserviceaccount.com`.
```
Comment (ytzur) [inline compute/admin_guide/vulnerability_management/vm_image_scanning.adoc]: Good, but please at the end of the sentence also add a link for reference to https://cloud.google.com/iam/docs/service-agents.

**Review by ytzur — APPROVED** [review]
LGTM, but please add the link to GCP docs for the service agent's email address.

File: compute/admin_guide/vulnerability_management/vm_image_scanning.adoc (hunk 1 above)
Comment (rcaballeromx) [reply compute/admin_guide/vulnerability_management/vm_image_scanning.adoc]: I've added the link again. Thanks for catching that.

**Review by Pubs-MV — APPROVED** [review]
Thank you!

**Pull request merged**